### ISSUE TITLE
Error: (SystemJS) ctorParameters.map is not a function 	TypeError: ctorParameters.map is not a function

### ISSUE THREAD

**Issue by rezoan** [oferh/ng2-completer#246]
Trying to use the ng2-completer but getting error 

Error: (SystemJS) ctorParameters.map is not a function
	TypeError: ctorParameters.map is not a function

my 
- package.json has ` "ng2-completer": "^1.5.0"`
- systemjs.config map has` 'ng2-completer': 'npm:ng2-completer/ng2-completer.umd.js'`
- App.module has 
```
1. `import { Ng2CompleterModule } from "ng2-completer"; `
2. `@NgModule imports has Ng2CompleterModule`
```

what i am doing wrong?

**Comment by oferh** [comment]
hard to say, need some more info like you system.js config

**Comment by kattianita** [comment]
Hello - I am getting the same error - Upgraded Angular from 2.0.0 to 2.1.0

<script>
        System.config({
            paths: {
                'npm:': 'node_modules/'
            },
            map: {
                app: 'app',
                '@angular/core': 'npm:@angular/core/bundles/core.umd.js',
                '@angular/common': 'npm:@angular/common/bundles/common.umd.js',
                '@angular/compiler': 'npm:@angular/compiler/bundles/compiler.umd.js',
                '@angular/platform-browser': 'npm:@angular/platform-browser/bundles/platform-browser.umd.js',
                '@angular/platform-browser-dynamic': 'npm:@angular/platform-browser-dynamic/bundles/platform-browser-dynamic.umd.js',
                '@angular/http': 'npm:@angular/http/bundles/http.umd.js',
                '@angular/forms': 'npm:@angular/forms/bundles/forms.umd.js',
                '@angular/router': 'npm:@angular/router/bundles/router.umd.js',
                'ng2-completer': 'npm:ng2-completer/ng2-completer.umd.js',
                'rxjs': 'npm:rxjs',
                
            },
            baseURL: '/',
            defaultJSExtensions: true,
            packages: {
                app: { defaultExtension: 'js' },
                rxjs: { defaultExtension: 'js' }
            }
        });
    </script>

**Comment by oferh** [comment]
can you try `'ng2-completer':             'npm:ng2-completer'`

See the demo [plunk](https://plnkr.co/edit/sVnfpBiEb5jBdtul4ls9?p=info) for a reference

**Comment by kattianita** [comment]
Now I get 

zone.js:232 Error: SyntaxError: Unexpected token <
	    at eval (<anonymous>)
	Evaluating http://localhost:2592/node_modules/ng2-completer.js
	Error loading http://localhost:2592/app/boot.js

**Comment by oferh** [comment]
did you try with Angular 2.1.0?

**Comment by kattianita** [comment]
Yes - and I get the error - (SystemJS) ctorParameters.map is not a function TypeError: ctorParameters.map is not a function

**Comment by oferh** [comment]
According to [this](https://stackoverflow.com/questions/41285889/ctorparameters-map-is-not-a-function-in-angular2-mdl)  it's an issue with Angular compiler-cli >= 2.3.0.
looks like the minimal supported version should be 2.3.0.

Maybe you can import from `ng2-completer/src` and not from `ng2-completer`.
